src/system/components/handler.clj

```diff
@@ -27,20 +28,28 @@
  [endpoint]
   (reduce-kv (fn [_ k v] (if (contains+? v :middleware) (reduced k) _)) {} (val endpoint)))
 
-(defrecord Handler []
+(def ^:private accepted-routing-lib #{:compojure :bidi})
```

Comment:
Please move this logic into component constructor. And simplify. For example, dispatch on a map.
`{:compojure #'compojure/routes
   :bidi (var (ns-resolve 'bidi.ring (symbol "make-handler")))}`